src/models/User.js

```diff
@@ -0,0 +1,31 @@
+import mongoose from "mongoose";
+
+const UserSchema = new mongoose.Schema(
+  {
+    type: String,
```

Comment:
To clarify on tyope and employeeType:
type: Customer, Employee, BrandOwner?
employeeType: Foreman, BranchManager?